modules/references/client/components/read-references/Meta.js

```diff
@@ -36,8 +36,6 @@ export default function Meta({ met, hostedMe, hostedThem, recommend }) {
 }
 
 Meta.propTypes = {
-  hostedMe: PropTypes.bool.isRequired,
-  hostedThem: PropTypes.bool.isRequired,
-  met: PropTypes.bool.isRequired,
+  interactions: PropTypes.object.isRequired,
```

Comment:
Could do something like this to describe the shape here:

https://github.com/Trustroots/trustroots/blob/07249bba5adfe650c9c62469617f8440cb137eeb/modules/users/client/users.prop-types.js#L1-L7

Totally also ok if not.

Reply:
oh, absolutely, also for experience object then, and experience pairs. I guess we can do it in a separate PR

Reply:
Yes, totes optional and fine in follow-ups. 👍